src/pages/components/testimonial/index.js

```diff
@@ -21,21 +21,21 @@ const Testimonial = ({ testimonialCaption, isMain = true, gallery = {} }) => {
     arrows: false,
     dots: false,
     infinite: true,
-    autoplay: false,
+    autoplay: true,
     speed: 500,
-    slidesToShow: gallery.length <= 3 ? gallery.length : 3.5,
-    lazyLoad: false,
+    slidesToShow: 4,
```

Comment:
if the images/data is only 2 items, it will display 4 
<img width="910" alt="image" src="https://user-images.githubusercontent.com/70557829/173300910-d2cb4f83-335d-41db-85dd-234b11cb15b8.png">